build/includes/terraform.mk

```diff
@@ -81,9 +85,21 @@ endif
 	GCP_CLUSTER_NAME=$(GCP_TF_CLUSTER_NAME) $(MAKE) gcloud-auth-cluster
 
 gcloud-terraform-destroy-cluster:
+	$(DOCKER_RUN) bash -c 'cd $(mount_path)/build/terraform/gke && \
+	 terraform destroy -target module.helm_agones.helm_release.agones -auto-approve && sleep 60 && terraform destroy -auto-approve'
```

Comment:
Does this mean this can now just be `terraform destroy` ?

Reply:
Yes, you are right I will make this change as well as website in a separate PR.